src/opengb/dram/quantity.cljc

```diff
@@ -263,10 +263,10 @@
             :USD_ft2_a         "USD/ft²/year"
             :m2                "m²"
             :GJ_m2_a           "GJ/m²/year"
-            :kWh_m2_a          "kWh/m²/year"
+            :kWh_m2_a          "ekWh/m²/year"
             :tCO2e_a           "tCO₂e/year"
             "m**2"             "m²"
-            "kWh/m**2/year"    "kWh/m²"
```

Comment:
I wasn't sure if I should just change how the unit was formatted for display, or change the unit's definition as well.

I decided to just change it here because that seemed less likely to break things.
This approach has downsides though.
- The unit used "behind-the-scenes" now differs even more from how it's displayed to users. This could be confusing if we have to troubleshoot issues with units.
- Following the above, this makes the way in which we format units for display even more idiosyncratic to GRID's usage. I'm beginning to think this whole function should be moved to GRID instead of being part of this library.
- This doesn't give us any way to distinguish between ekWh/m**2/year and kWh/m**2/year behind-the-scenes. That isn't something we need right now, but if we do ever need it, it will be slightly more work to separate the two.

These seem to be minor, but I think we should be aware of them.

Reply:
The team discussed this today and decided that we'll go with the cosmetic change for now, and plan to do the more robust change in the future.